addons/website/static/src/snippets/s_accordion/accordion.js

```diff
@@ -0,0 +1,24 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+export class Accordion extends Interaction {
+    static selector = ".s_accordion";
+
+    start() {
+        // Make the accordion button text editable
+        const buttonEls = this.el.querySelectorAll(".accordion-button");
+        buttonEls.forEach(buttonEl => {
```

Comment:
```suggestion
        buttonEls.forEach((buttonEl) => {
```

To make linter happy